erlang/crypto_tests/sign.erl

```diff
@@ -0,0 +1,42 @@
+-module(sign).
+
+-export([cmd/0]).
+
+-include_lib("stdlib/include/assert.hrl").
+-include_lib("public_key/include/public_key.hrl"). 
+
+read_key(private, Name) ->
+    {ok, PemBin} = file:read_file(Name),
+    [RSAEntry] = public_key:pem_decode(PemBin),
+    Key = public_key:pem_entry_decode(RSAEntry),
+    Key;
+
+read_key(public, Name) ->
+    {ok, Bin} = file:read_file(Name),
+    [{Key, _}] = public_key:ssh_decode(Bin, public_key),
+    % [{Key, _}] <- spent like literally an hour figuring that one out, so stupid ಠ_ಠ
+    Key.
+
+
+cmd() ->
+    % HASH
+
+    Message = "hi",
+    BinMessage = list_to_binary(Message),
+    MessageHash = crypto:hash(sha256, BinMessage),
+    L = binary_to_list(MessageHash),
+    io:format("string ~p hash_sz: ~p hash: ~p ~n", [Message, length(L), L]),
+
+    % RSA PRIVATE
+    Key = read_key(private, "rsa_private"),
```

Comment:
shouldn't this use my_crypto::read_file_key?

Reply:
unrelated file, i'll move it to a different folder

Reply:
this was just me messing with some stuff before creating my_crypto library

Reply:
👍👍👍